examples/karpenter/main.tf

```diff
@@ -78,6 +78,16 @@ module "eks" {
   cluster_version                = local.cluster_version
   cluster_endpoint_public_access = true
 
+  cluster_addons = {
+    kube-proxy = {}
+    vpc-cni    = {}
+    coredns = {
+      configuration_values = jsonencode({
+        computeType = "Fargate"
```

Comment:
this is awesome

Reply:
Hi ! Sorry to ask here, but why is this needed ?
(and I agree its awesome)

Reply:
@auyer until now it was super hard to run this managed addon on fargate since it would default to ec2 and offered no option to change that.
One would have to run complex script to modify the deployment and that would complicate upgrades.